lib/mongo/crypt/binding.rb

```diff
@@ -185,31 +174,30 @@ class Binding
       # A callback to the mongocrypt log function
       # Set a custom log callback with the mongocrypt_setopt_log_handler method
       #
-      # @param [ Symbol ] level The log level; possible values defined by the
+      # param [ Symbol ] level The log level; possible values defined by the
```

Comment:
I think it is beneficial to have the callbacks' api docs also be rendered, since all of the work to document them has already been done. How about adding a note either per method or one time in the class docstring to the effect of "the entries ending in `_fn` are FFI callback definitions and not methods defined on this class"?

Reply:
Sounds good!